DESCRIPTION

```diff
@@ -18,7 +18,9 @@ BugReports:
 Depends: 
     R (>= 3.5),
 Imports: 
-    R6
+    R6,
+    tlf (>= 0.1.0),

```

Comment:
Yes, 
It still fails to install on github while my devtools::check() worked.
I think it is because of the Remote option that I can't add to description without writing directly on the DESCRIPTION file:
https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html

Reply:
Should we simply install it in the build script instead? 
similar to what I do with rCLR (which will need to be installed to for the build to pass) and which i don't see here
Suggestion:
in build script, we find a place to insta
rClr
ospsuite
tlf

Woudn't that work?

Reply:
I think so. Is the build script something to set in appveyor ?
If at some point the packages are uploaded on CRAN, we won't need this workaround anymore

Reply:
TLF but OSPSuite won't because of the binaries we are installing

Reply:
Can you send me the workflow to add rClr, ospsuite and tlf into the build script as you did for ospsuite ?
Thanks

Reply:
@msevestre 
Can you send me the steps or the devtools functions that manage the installation of these packages in the build script ?